pandas/_libs/tslibs/timestamps.pyi

```diff
@@ -117,12 +121,12 @@ class Timestamp(datetime):
         second: int = ...,
         microsecond: int = ...,
         tzinfo: _tzinfo | None = ...,
-        *,
         fold: int = ...,
     ) -> datetime: ...
     def astimezone(self: _S, tz: _tzinfo | None = ...) -> _S: ...
     def ctime(self) -> str: ...
-    def isoformat(self, sep: str = ..., timespec: str = ...) -> str: ...
+    # error: Signature of "isoformat" incompatible with supertype "datetime"
+    def isoformat(self, sep: str = ...) -> str: ...  # type: ignore[override]
```

Comment:
the implementation does not accept `timespec`

Reply:
hmm i thought we recently changed this

Reply:
Yes, that seems to have changed since Nov 6 :)

I will look through the other cases that don't match datetime and correct them if necessary in a new PR.